Tools/wol/wol-nas.10s.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+#  <xbar.title>Wake on LAN (WoL)</xbar.title>
+#  <xbar.version>v1.0</xbar.version>
+#  <xbar.author>Michael R.</xbar.author>
+#  <xbar.author.github>The-Michael-R</xbar.author.github>
+#  <xbar.desc>Small xbar plugin to wake one server/computer using a WoL packet. Needs https://github.com/jpoliv/wakeonlan (brew or place the file wakeonlan to /urs/local/bin</xbar.desc>
+#  <xbar.image>https://github.com/The-Michael-R/xbar-plugins/Tools/wol/wol_xbar.png</xbar.image>
```

Comment:
This image is not found https://github.com/The-Michael-R/xbar-plugins/Tools/wol/wol_xbar.png